packages/utils/date.js

```diff
@@ -139,13 +68,17 @@ export function parsePOSLocalDatetime(dateString) {
       offsetMin: "00"
     */
 
-    const { offsetHour } = window.Clock.getCurrentTimeZone();
-    return parse({
-      date: dateString,
-      utc: true,
-      offsetHour: Number(offsetHour),
-    });
+    const { offsetHour, offsetMin } = window.Clock.getCurrentTimeZone();
+    if (dateFormat) {
+      return format(
+        utcToZonedTime(dateString, `${offsetHour}${offsetMin}`),
+        dateFormat,
+      );
+    }
+    return utcToZonedTime(dateString, `${offsetHour}${offsetMin}`);
```

Comment:
O segundo parâmetro não deveria ser a região ? `America/Sao_Paulo`

Reply:
A região não é suportada por browsers antigos, somente o número do timezone, essas regiões são para browsers modernos usando a org IANA.

Reply:
Dependency free IANA time zone support is implemented via the Intl API to keep actual time zone data out of code bundles. Modern browsers all support the necessary features, and for those that don't a polyfill can be used.

Reply:
If you do not wish to use a polyfill the time zone option can still be used, but only with time zone offsets such as '-0200' or '+04:00' and not IANA time zone names.